### ISSUE TITLE
Meraki Lab not well documented preventing attendees complete the lab

### ISSUE THREAD

**Issue by marivalis123** [CiscoDevNet/devnet-express-dna-issues#107]
https://developer.cisco.com/learning/devnet-express/dnav4-track/dnav3-meraki/meraki-07-webhooks-dna/step/2
Lab describe following commands:
(venv)$ cd ~/code/meraki-code/meraki-webhooks
(venv)$ python webhookreceiver.py -s webbie

but the directory should be ~/code/dna-dne-code/intro-meraki. 
Not described, but before running WebHook Server, we need to apply the command "pip install -r requirements.txt"
Otherwise, the wehookreceiver.py program will fail.

We need to also describe how to run the Cloud Simulator. It should be similar, going to the folder meraki_cloud_simulator, install requirements.txt and launch the python script. But this is not described.

**Comment by Nockternal** [comment]
Agreed the labs are not clear to follow with very little time to complete. and people are not responding to questions when things go wrong. 
also each separate section is not being explained to referenced to external site where the documentation is.
